test/work_item_repository.go

```diff
@@ -76,6 +76,26 @@ type WorkItemRepository struct {
 		result1 *app.WorkItem
 		result2 error
 	}
+	GetCountsPerIterationStub        func(ctx context.Context, spaceID uuid.UUID) (map[string]workitem.WICountsPerIteration, error)
```

Comment:
I ran `counterfeit` again to regenerate `test/work_item_repository.go` and this was part of the addition.